Mathlib/Tactic/TacticAnalysis.lean

```diff
@@ -0,0 +1,371 @@
+/-
+Copyright (c) 2025 Lean FRO LLC. All rights reserved.
```

Comment:
Oh dear, we are inconsistent about this, but I think `Lean FRO, LLC` is preferred.

Reply:
```suggestion
Copyright (c) 2025 Lean FRO, LLC. All rights reserved.
```